tsconfig.json

```diff
@@ -1,6 +1,16 @@
 {
-	"extends": "./config/tsconfig.base",
-	"exclude": [
-		"node_modules/**"
-	]
-}
\ No newline at end of file
+	"extends": "@sofie-automation/code-standard-preset/ts/tsconfig.lib",
```

Comment:
this should extend "./tsconfig.build.json", so that compilerOptions can be defined once and be used by both versions of the file